test_scripts/RC/CLIMATE_RADIO/GetInteriorVehicleDataConsent/011_TIMED_OUT_after_default_timeout.lua

```diff
@@ -39,11 +40,11 @@ local function rpcHMIRespondAfterDefaultTimeout()
         commonRC.getHMIConnection():SendError(data.id, data.method, "TIMED_OUT", "info")
         EXPECT_HMICALL(commonRC.getHMIEventName(pRPC1)):Times(0)
       end
-      RUN_AFTER(hmiRespond, 11000)
+      RUN_AFTER(hmiRespond, commonRC.defaultTimeoutWithCompensation + 1000)
     end)
 
   commonRC.getMobileSession(2):ExpectResponse(cid1, { success = false, resultCode = "GENERIC_ERROR" })
-  commonRC.wait(12000)
+  commonRC.wait(13000)
```

Comment:
I propose to perform such update because this value is depended on `defaultTimeoutWithCompensation`
```suggestion
  commonRC.wait(commonRC.defaultTimeoutWithCompensation + 2000)
```